lib/rules/lint-no-dom-events.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+const ERROR_MESSAGE = 'Use Ember actions instead of DOM events.';
+
+module.exports = class NoDomEvents extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        const domEventAttribute = node.attributes.find(isDomEventAttributeWithAction);
```

Comment:
I could be wrong, but doesn't `[].find` return the first item? I think we would actually want to `[].filter` and report _all_ of them...

Imagine:

```hbs
<button onclick={{action 'foo'}} ondblclick={{action 'bar'}}></button>
```

We would want both of those reported, not just the first one.

Reply:
Good point. Switched to `filter` and added a test for this.